Modules/Numerics/Eigen/wrapping/itkEigenAnalysis2DImageFilter.wrap

```diff
@@ -1,5 +1,11 @@
 itk_wrap_class("itk::EigenAnalysis2DImageFilter" POINTER)
+  # Only vectors with 2 components are valid. Temporarily change the global vector dims list.
+  set(BACKUP_ITK_WRAP_VECTOR_COMPONENTS "${ITK_WRAP_VECTOR_COMPONENTS}")
```

Comment:
Perhaps replace / supplement `BACKUP_` with something that includes `EIGEN_ANALYSIS_2D_IMAGE_FILTER_`.